egs/wsj/s5/utils/prepare_lang.sh

```diff
@@ -143,6 +149,15 @@ if [[ ! -z $phone_symbol_table ]]; then
       print "Phone appears in the lexicon but not in the provided phones.txt: "$x; exit 1; }}}' || exit 1;
 fi
 
+# In case there are extra word-level disambiguation symbols we need
+# to make sure that all symbols in the provided file are valid.
+if [[ ! -z $extra_word_disambig_syms ]]; then
+  if ! utils/validate_disambig_sym_file.pl --extra-disambig-syms $extra_word_disambig_syms; then
+    echo "Validation of disambiguation file \"$extra_word_disambig_syms\" failed."
```

Comment:
please start echo statements with `"$0:` so it's clear which program produced the errror.